libs/components/phone-field/src/lib/modules/phone-field/phone-field.component.ts

```diff
@@ -114,7 +114,7 @@ export class SkyPhoneFieldComponent implements OnDestroy, OnInit {
    * @default true
    */
   @Input()
-  public allowExtensions = true;
+  public allowExtensions: boolean | undefined = true;
```

Comment:
Since this input defaults to `true`, we might need to implement a getter/setter. See: https://dev.azure.com/blackbaud/Products/_wiki/wikis/Products.wiki/3239/Components?anchor=boolean-inputs

Reply:
fixed in: [add access modifiers to inputs](https://github.com/blackbaud/skyux/pull/740/commits/34e778b786c8b4e4ea48c706068ab13dd0304bef)